napari/_qt/qt_viewer.py

```diff
@@ -35,6 +36,17 @@
 from .._vispy import create_vispy_visual
 
 
+class KeyModifierFilterSceneCanvas(SceneCanvas):
+    """SceneCanvas which overrides default vispy behaviour for events with key modifiers.F
+
+    """
```

Comment:
PEP8/PEP257: one-liners should fit in one 79c line. 
    
```suggestion
    """SceneCanvas overriding VisPy when mouse wheel events have modifiers."""
```